lib/utils/patterns.ts

```diff
@@ -18,12 +18,12 @@ import _ = require('lodash');
 import Promise = require('bluebird');
 import form = require('resin-cli-form');
 import visuals = require('resin-cli-visuals');
-import ResinSdk = require('resin-sdk');
+import ResinSdk = require('resin-sdk-preconfigured');
```

Comment:
Here too - we only ever look at device & app names & device types, which will still work totally fine with MC.

Reply:
I'm not sure of that, device.name was renamed to device.device_name for example.

Reply:
Ah, good spot! In that case I expect you can probably just follow the TS errors and rename the field in a couple of places below to sort it, but otherwise I'm not too bothered by moving to preconfigured here if that's tricky for some reason.

We should probably quickly manually test a couple of the commands that use this either way, since it's currently v7, so MC or preconfigured would both require a major version bump in one direction or another.